CONTRIBUTING.md

```diff
@@ -30,52 +52,96 @@ For more information specific to how a quarto site works, and for examples of wh
 - building locally before raising a pull request
 
 -->
```

Comment:
This is now showing the comment in the preview of the file, was this intentional? 

![image](https://github.com/dfe-analytical-services/analysts-guide/assets/52536248/55a41d5d-ffca-494c-8b22-63b36d6b7d48)